openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -0,0 +1,99 @@
+from rest_framework.response import Response
+from rest_framework import viewsets, status
+from rest_framework.decorators import permission_classes, authentication_classes
+from rest_framework.filters import OrderingFilter
+from rest_framework.authentication import SessionAuthentication
+
+from django.contrib.auth.models import User
+from student.models import UserAttribute
+from django.db.models.query_utils import Q
+
+from .serializers import UserSerializer
+
+from .permissions import CanAccessPakXAdminPanel
+
+from .constants import GROUP_TRAINING_MANAGERS, ADMIN, TRAINING_MANAGER
+
+
+@authentication_classes([SessionAuthentication])
+@permission_classes([CanAccessPakXAdminPanel])
```

Comment:
decorator is mostly used for functional view. for class based view please use https://www.django-rest-framework.org/api-guide/authentication/